lib/os_places_api/location_builder.rb

```diff
@@ -4,12 +4,12 @@ def initialize(result)
       @result = result
     end
 
-    def build_location
-      Location.new(postcode: @result.dig("DPA", "POSTCODE"),
-                   address: @result.dig("DPA", "ADDRESS"),
-                   latitude: @result.dig("DPA", "LAT"),
-                   longitude: @result.dig("DPA", "LNG"),
-                   local_custodian_code: @result.dig("DPA", "LOCAL_CUSTODIAN_CODE"))
+    def build_location(dataset)
+      Location.new(postcode: @result.dig(dataset, "POSTCODE"),
```

Comment:
Looking again, I think we can actually remove this parameter - no need for us to store the `postcode` in the JSON when we already have a sanitised & normalised `postcode` field in the same row in the database. That will also require a tweak to the Pact tests though 😅 

It feels like a separate piece of work, except that we're only having to do this work because of the missing `POSTCODE` field in the LPI results, so also fine for this work to be bundled into this PR.

Reply:
The OS API response is stored raw in the db. The Location instance is used for building informations about a specific postcode. But I agree we don't need the postcode from the api response as long as we have the sanitised one. Also, `Locations.postcode` is not used anywhere yet so we can remove it.

Reply:
Ah yes, I misremembered!